cli/cmd/compliance_gcp.go

```diff
@@ -32,10 +32,46 @@ import (
 )
 
 var (
+	// complianceGcpListCmd represents the list sub-command inside the gcp command
+	complianceGcpListCmd = &cobra.Command{
+		Use:   "list",
+		Short: "list gcp projects and organizations",
+		Long:  `List all GCP projects and organization IDs.`,
+		RunE: func(_ *cobra.Command, args []string) error {
+			var (
+				gcpAccounts   []gcpAccount
+				response, err = cli.LwApi.Integrations.ListGcpCfg()
+				rows          [][]string
+			)
+
+			if err != nil {
+				return errors.Wrap(err, "unable to list gcp projects/organizations")
+			}
+
+			for _, gcp := range response.Data {
+				gcpIntegration := gcpAccount{}
+				// if organization account, fetch the project ids
+				if gcp.Data.IDType == "ORGANIZATION" {
+					gcpAccounts = append(gcpAccounts, getGcpAccounts(gcpIntegration.OrganizationID)...)
+				} else if !containsDuplicateProjectID(gcpAccounts, gcp.Data.ID) {
+					gcpIntegration = gcpAccount{OrganizationID: "n/a", ProjectID: gcp.Data.ID}
+					gcpAccounts = append(gcpAccounts, gcpIntegration)
+				}
+			}
+
+			for _, gcp := range gcpAccounts {
+				rows = append(rows, []string{gcp.ProjectID, gcp.OrganizationID})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"Project ID", "Organization ID"}, rows))
```

Comment:
nit: I feel that it is better to show ORG ID and then Project ID like this
```suggestion
			cli.OutputHuman(renderSimpleTable([]string{"Organization ID", "Project ID"}, rows))
```

why? Because of 1) the hierarchy orgs can have projects and 2) the user can copy and paste the like to download the compliance report which asks for:
```
lacework compliance google get-report <organization_id> <project_id>
```